lib/parsers/gjs-parser.js

```diff
@@ -50,10 +50,11 @@ function findVarInParentScopes(scopeManager, nodePath, name) {
     }
     path = path.parentPath;
   }
+  const currentScope = findParentScope(scopeManager, nodePath);
   if (!scope) {
-    return { scope: findParentScope(scopeManager, nodePath) };
```

Comment:
needed to fix the current scope, as the references should be added in the scope where the references are and not where the variable definition is